src/main/java/com/socrata/api/Soda2Producer.java

```diff
@@ -76,6 +78,7 @@ public static final Soda2Producer newProducerWithRequestId(final String url, Str
     public Soda2Producer(HttpLowLevel httpLowLevel)
     {
         super(httpLowLevel);
+        factory = new JsonFactory();
```

Comment:
Again, I need some help running the tests. When I try to run them, I get a whole lot of these failures in my terminal

```
[info] Test com.socrata.DatesTest.testFloatingDateSoql started
[error] Test com.socrata.DatesTest.testFloatingDateSoql failed: Authentication failed: Invalid username or password
[error]     at com.socrata.api.HttpLowLevel.processErrors(HttpLowLevel.java:687)
[error]     at com.socrata.api.HttpLowLevel.queryRaw(HttpLowLevel.java:448)
[error]     at com.socrata.api.Soda2Base.query(Soda2Base.java:62)
[error]     at com.socrata.api.Soda2Consumer.query(Soda2Consumer.java:111)
[error]     at com.socrata.DatesTest.testFloatingDateSoql(DatesTest.java:52)
```
I didn't see anything in the README about how to avoid this. There are credentials in TestConfig.properties, but they don't appear to be working